jobs/data/repolist.txt

```diff
@@ -48,11 +49,13 @@ jorive/coreclr branch=dev/baseline-with-new-api server=dotnet-ci2 definitionScri
 dotnet/coreclr branch=release/1.0.0 server=dotnet-ci
 dotnet/coreclr branch=release/1.1.0 server=dotnet-ci
 dotnet/coreclr branch=release/2.0.0 server=dotnet-ci
+dotnet/coreclr branch=release/uwp6.0 server=dotnet-ci
 dotnet/coreclr branch=dev/unix_test_workflow server=dotnet-ci
 dotnet/corefx branch=master server=dotnet-ci
 dotnet/corefx branch=release/1.0.0 server=dotnet-ci
 dotnet/corefx branch=release/1.1.0 server=dotnet-ci
 dotnet/corefx branch=release/2.0.0 server=dotnet-ci
+dotnet/corefx branch=release/uwp6.0 server=dotnet-ci
```

Comment:
I don't have context on how we want the UWP work to run, so just make sure that we do want them to use `dotnet-ci` and not the pipeline jobs like corefx release/2.0 does (`dotnet-ci3`)